service/routes/api.js

```diff
@@ -68,19 +68,18 @@ router.get(/^\/v2\/polyfill(\.\w+)(\.\w+)?/, (req, res) => {
 		metrics.counter('useragentcount.'+polyfillio.normalizeUserAgent(uaString).replace(/^(.*?)\/(\d+)(\..*)?$/, '$1.$2')).inc();
 	}
 
-	polyfillio.getPolyfillString(params).then(op => {
-		if (warnings.length) {
-			op = '/* WARNINGS:\n\n- ' + warnings.join('\n- ') + '\n\n*/\n\n' + op;
-		}
+	res.set('Content-Type', contentTypes[fileExtension]+';charset=utf-8');
+	res.set('Access-Control-Allow-Origin', '*');
+
+	const outputStream = polyfillio.getPolyfillString(params);
+	outputStream.pipe(res, {end: false});
+	outputStream.on('end', () => {
 		if (req.query.callback && req.query.callback.match(/^[\w\.]+$/)) {
```

Comment:
Right, well, thanks for that :-)

I think restricting callback names to ASCII and allowing use of reserved words is fine.  If someone wants to `alert` the result of the jsonp call, I have no objection!